Fix:update .bit name in nft-collection and bridge token

JeffreyMa597 opened the pull request:
What problem does this PR solve?
------

## 1 
1-1、 support searching ends with `.bit`
1-2、the .bit domain tag on the account page should be a `link` to .bit account page

<img width="500" alt="image" src="https://user-images.githubusercontent.com/22511289/211251396-7a6517c4-55cc-4792-ac61-5f07bfebf33d.png">

## 2
2-1、 showing magickbase.bit on hovering .bit icon is redundant, now showing text `.bit Name`

<img width="500" alt="image" src="https://user-images.githubusercontent.com/22511289/211251483-3471b309-b289-4326-a23b-cf842314be6b.png">

## 3
3-1、 token/holder list, magickbase.bit is not displayed ( `not include in this PR`)
https://v1.gwscan.com/token/133?tab=holders&id=133&page=13
3-2、 nft-collection, in the inventory, owner is not displayed by .bit name https://v1.gwscan.com/nft-collection/0x73c0e2a4b765ed81f63759d1fcf2c4dcd2925f4a?tab=inventory
3-3、nft-item, owner is not displayed with .bit name
https://v1.gwscan.com/nft-item/0x73c0e2a4b765ed81f63759d1fcf2c4dcd2925f4a/2375

<img width="500" alt="image" src="https://user-images.githubusercontent.com/22511289/211251429-0eeb9228-7cc5-440a-a8f6-5a076560fcbd.png">


Ref: https://github.com/Magickbase/godwoken_explorer/issues/1170

Check List
------

### Test
e2e Test
### Task
none

FrederLu commented:
1、Now click to jump, but the text color is wrong.
<img width="909" alt="image" src="https://user-images.githubusercontent.com/25676576/211287099-a18a5853-847c-4e17-aa7e-bf44fcc5502e.png">

2、The address whose .bit alias has been set needs to display the alias.
https://godwoken-explorer-oklw14yhl-magickbase.vercel.app/token/84440?tab=holders
<img width="1257" alt="image" src="https://user-images.githubusercontent.com/25676576/211289340-3e74ec6e-c985-4900-a777-47d7f533bb3f.png">

JeffreyMa597 commented:
> 1、Now click to jump, but the text color is wrong. <img alt="image" width="909" src="https://user-images.githubusercontent.com/25676576/211287099-a18a5853-847c-4e17-aa7e-bf44fcc5502e.png">

updated.

FrederLu commented:
<img width="670" alt="image" src="https://user-images.githubusercontent.com/25676576/212290657-cfeb31c7-c7b3-47d7-b94e-867592ef2766.png">
You can see the corresponding data display below.

1、The corresponding data is not displayed.
https://godwoken-explorer-lf9snmtks-magickbase.vercel.app/nft-collection/0x0772997bfde614faa246b3cc98358f459d7911f7?tab=holders
<img width="1368" alt="image" src="https://user-images.githubusercontent.com/25676576/212289970-1d6721af-6cb9-4226-a640-06ae963e829a.png">
2、No alias information displayed yet.
https://godwoken-explorer-lf9snmtks-magickbase.vercel.app/nft-collection/0x0772997bfde614faa246b3cc98358f459d7911f7?tab=inventory
<img width="1321" alt="image" src="https://user-images.githubusercontent.com/25676576/212290511-19bf2123-aa0e-479a-bd44-d76e124a3cf4.png">
https://godwoken-explorer-lf9snmtks-magickbase.vercel.app/nft-item/0x0772997bfde614faa246b3cc98358f459d7911f7/18
<img width="1171" alt="image" src="https://user-images.githubusercontent.com/25676576/212290979-b1454353-7488-439d-b7da-dc8bedaed45d.png">

JeffreyMa597 commented:
> 1、The corresponding data is not displayed. https://godwoken-explorer-lf9snmtks-magickbase.vercel.app/nft-collection/0x0772997bfde614faa246b3cc98358f459d7911f7?tab=holders 

fixed.

2、No alias information displayed yet. https://godwoken-explorer-lf9snmtks-magickbase.vercel.app/nft-collection/0x0772997bfde614faa246b3cc98358f459d7911f7?tab=inventory 

waiting for the update from our endpoint.

FrederLu commented:
https://godwoken-explorer-2c6ln4cjd-magickbase.vercel.app/token/84440?tab=holders
<img width="1278" alt="image" src="https://user-images.githubusercontent.com/25676576/212903253-a233c569-d670-4a94-b5bf-94e1903b432c.png">
Just to make sure, there are still some changes to be made here.

JeffreyMa597 commented:
> https://godwoken-explorer-2c6ln4cjd-magickbase.vercel.app/token/84440?tab=holders <img alt="image" width="1278" src="https://user-images.githubusercontent.com/25676576/212903253-a233c569-d670-4a94-b5bf-94e1903b432c.png"> Just to make sure, there are still some changes to be made here.

@FrederLu  @Naupio I wondered whether I can submit other changes first with this PR except the changes in the `holders` tab. We still waiting for the update from our endpoint to change the `http` request to `Graphql` request. Thank you.

FrederLu commented:
@JeffreyMa597  The following questions need to be confirmed:
https://godwoken-explorer-9mgu7gvzq-magickbase.vercel.app/nft-item/0x0772997bfde614faa246b3cc98358f459d7911f7/18
1、On the above page, changing the page size will distort the icons.
<img width="1023" alt="image" src="https://user-images.githubusercontent.com/25676576/217771515-d06affa3-4252-449d-b26d-4c1ff72b3326.png">

2、After `Owner` is changed to an alias, the alignment with `TokenId` needs to be fine-tuned.
https://godwoken-explorer-9mgu7gvzq-magickbase.vercel.app/nft-collection/0x0772997bfde614faa246b3cc98358f459d7911f7?tab=inventory
<img width="919" alt="image" src="https://user-images.githubusercontent.com/25676576/217772740-dfce22ea-c44f-4245-b1fa-9a86ff8b5ac7.png">

JeffreyMa597 commented:
> @JeffreyMa597 The following questions need to be confirmed: https://godwoken-explorer-9mgu7gvzq-magickbase.vercel.app/nft-item/0x0772997bfde614faa246b3cc98358f459d7911f7/18 1、On the above page, changing the page size will distort the icons. 

Fixed.
<img width="500" alt="image" src="https://user-images.githubusercontent.com/22511289/220400807-3ad3efae-ed6c-4f6a-bca0-3c784a450444.png">
 


> 2、After `Owner` is changed to an alias, the alignment with `TokenId` needs to be fine-tuned. https://godwoken-explorer-9mgu7gvzq-magickbase.vercel.app/nft-collection/0x0772997bfde614faa246b3cc98358f459d7911f7?tab=inventory 

Fixed. 
<img width="500" alt="image" src="https://user-images.githubusercontent.com/22511289/220401065-0114aabc-0ca3-459a-b00e-3187d9e2340c.png">


> 3、There are addresses with aliases under different tabs on the same page, but the display is abnormal. (A)https://godwoken-explorer-9mgu7gvzq-magickbase.vercel.app/multi-token-item/0xfa4d45c9b89d8b8025ca7a5c8aab15ba03f32db9/4?tab=holders 

Fixed.
<img width="500" alt="image" src="https://user-images.githubusercontent.com/22511289/220407472-b0c7500b-dc89-46ec-b5cd-c93afb483cf7.png">

 
> (B)https://godwoken-explorer-9mgu7gvzq-magickbase.vercel.app/multi-token-item/0xfa4d45c9b89d8b8025ca7a5c8aab15ba03f32db9/4?tab=inventory 

Fixed.
<img width="500" alt="image" src="https://user-images.githubusercontent.com/22511289/220411275-fa8801dd-c655-4a7a-bd86-2f0ef98c7f47.png">

 
> 4、There is no data in holders. https://godwoken-explorer-9mgu7gvzq-magickbase.vercel.app/multi-token-collection/0xfa4d45c9b89d8b8025ca7a5c8aab15ba03f32db9?tab=holders <img alt="image" width="500" src="https://user-images.githubusercontent.com/25676576/217784680-aceb53b8-fdc5-446b-b81b-3eef7ac020fc.png">

Will fix in another PR.

FrederLu commented:
The size of the picture on the mobile terminal is not adapted.
Basemap：https://godwoken-explorer-8afi0eh63-magickbase.vercel.app/nft-item/0x0772997bfde614faa246b3cc98358f459d7911f7/18
Picture ：https://godwoken-explorer-8afi0eh63-magickbase.vercel.app/nft-item/0xfec8e80db5d57b23fad94b828c88d716d3c6226b/10
<img width="689" alt="image" src="https://user-images.githubusercontent.com/25676576/222087188-4057b842-4398-4986-a73e-64b2fd3502c9.png">
<img width="354" alt="image" src="https://user-images.githubusercontent.com/25676576/222087658-318884bc-1a2b-4032-9a95-99aeeb4cdd70.png">

Keith-CY commented:
> The size of the picture on the mobile terminal is not adapted. Basemap：[godwoken-explorer-8afi0eh63-magickbase.vercel.app/nft-item/0x0772997bfde614faa246b3cc98358f459d7911f7/18](https://godwoken-explorer-8afi0eh63-magickbase.vercel.app/nft-item/0x0772997bfde614faa246b3cc98358f459d7911f7/18) Picture ：[godwoken-explorer-8afi0eh63-magickbase.vercel.app/nft-item/0xfec8e80db5d57b23fad94b828c88d716d3c6226b/10](https://godwoken-explorer-8afi0eh63-magickbase.vercel.app/nft-item/0xfec8e80db5d57b23fad94b828c88d716d3c6226b/10) <img alt="image" width="689" src="https://user-images.githubusercontent.com/25676576/222087188-4057b842-4398-4986-a73e-64b2fd3502c9.png"> <img alt="image" width="354" src="https://user-images.githubusercontent.com/25676576/222087658-318884bc-1a2b-4032-9a95-99aeeb4cdd70.png">

It could be tracked by another issue because the size of an NFT image is not related to .bit

FrederLu commented:
> 
OK.
This pr modified the .bit avatar problem. If it is confirmed that it has nothing to do with this pr, open another issue to track.

FrederLu commented:
> > The size of the picture on the mobile terminal is not adapted. Basemap：[godwoken-explorer-8afi0eh63-magickbase.vercel.app/nft-item/0x0772997bfde614faa246b3cc98358f459d7911f7/18](https://godwoken-explorer-8afi0eh63-magickbase.vercel.app/nft-item/0x0772997bfde614faa246b3cc98358f459d7911f7/18) Picture ：[godwoken-explorer-8afi0eh63-magickbase.vercel.app/nft-item/0xfec8e80db5d57b23fad94b828c88d716d3c6226b/10](https://godwoken-explorer-8afi0eh63-magickbase.vercel.app/nft-item/0xfec8e80db5d57b23fad94b828c88d716d3c6226b/10) <img alt="image" width="689" src="https://user-images.githubusercontent.com/25676576/222087188-4057b842-4398-4986-a73e-64b2fd3502c9.png"> <img alt="image" width="354" src="https://user-images.githubusercontent.com/25676576/222087658-318884bc-1a2b-4032-9a95-99aeeb4cdd70.png">
> 
> It could be tracked by another issue because the size of an NFT image is not related to .bit

OK.
This pr modified the .bit avatar problem. If it is confirmed that it has nothing to do with this pr, open another issue to track.

Keith-CY commented on components/MultiTokenInventoryList/index.tsx:
```diff
@@ -14,7 +15,7 @@ type InventoryListProps = {
       contract_address_hash: string
       counts: string
-      owner?: string
+      owner?: Record<string, any>
```
Is the type of value unpredicatable?

Keith-CY commented on components/MultiTokenInventoryList/index.tsx:
```diff
@@ -120,51 +121,58 @@ const MultiTokenInventoryList: React.FC<InventoryListProps> = ({ inventory, view
                   <span>{t('owner')}</span>
-                  <HashLink href={`/account/${item.owner}`} label={item.owner} />
+                  <Address address={eth_address} domain={bit_alias} />
+                  {/* <HashLink href={`/account/${item.owner}`} label={item.owner} /> */}
```
Unnecessary comment could be removed

FrederLu commented:
> https://godwoken-explorer-2c6ln4cjd-magickbase.vercel.app/token/84440?tab=holders <img alt="image" width="1278" src="https://user-images.githubusercontent.com/25676576/212903253-a233c569-d670-4a94-b5bf-94e1903b432c.png"> Just to make sure, there are still some changes to be made here.

<img width="1072" alt="image" src="https://user-images.githubusercontent.com/25676576/223634779-461291c0-e0d0-49ad-816d-c6ee87068a7d.png">
Except that the holder data interface in pr has not been replaced, so the .bit alias is not displayed, and will be modified in other pr later.
Everything else has been fixed.
